### PR TITLE
All Describe command unified get data from metad

### PR DESCRIPTION
**Pull request by laura-ding**
1、Describe Tag and Edge from metad
2、"Use space"  get spaceId from metad
3、Modify DescribeSpace

### CODE REVIEW

**Comment by laura-ding** [comment]
jenkins go

File: src/meta/processors/partsMan/GetSpaceProcessor.cpp [hunk 1]
```diff
@@ -11,17 +11,24 @@ namespace meta {
 
 void GetSpaceProcessor::process(const cpp2::GetSpaceReq& req) {
     folly::SharedMutex::ReadHolder rHolder(LockUtils::spaceLock());
-    auto spaceId = req.get_space_id();
+    auto spaceRet = getSpaceId(req.get_space_name());
+    if (!spaceRet.ok()) {
+        resp_.set_code(to(std::move(spaceRet.status())));
```
Comment (dutor) [inline src/meta/processors/partsMan/GetSpaceProcessor.cpp]: use `std::move(spaceRet).status()` instead.

BTW. Please change `to` to passing by const reference. Then here `std::move` is no longer needed.

File: src/meta/test/ProcessorTest.cpp [hunk 2]
```diff
@@ -316,7 +330,7 @@ TEST(ProcessorTest, CreateEdgeTest) {
         ASSERT_EQ(cpp2::ErrorCode::E_EXISTED, resp.code);
     }
     {
-        // create same name edge in diff spaces
+        // Create same name edge in diff spaces
```
Comment (dutor) [inline src/meta/test/ProcessorTest.cpp]: 👍

File: src/meta/processors/partsMan/GetSpaceProcessor.cpp (hunk 1 above)
Comment (laura-ding) [reply src/meta/processors/partsMan/GetSpaceProcessor.cpp]: Good catch! 👍

File: src/graph/DescribeEdgeExecutor.cpp [hunk 3]
```diff
@@ -25,35 +25,43 @@ Status DescribeEdgeExecutor::prepare() {
...
+
+        resp_->set_rows(std::move(rows));
+        DCHECK(onFinish_);
+        onFinish_();
+    };
+
+    auto error = [this] (auto &&e) {
+        LOG(ERROR) << "Exception caught: " << e.what();
+        DCHECK(onError_);
+        onError_(Status::Error("Internal error"));
```
[the remark this answers is not in the document]
Comment (laura-ding) [reply src/graph/DescribeEdgeExecutor.cpp]: The error message is internal, I think user can’t need konw it directly, but need to Unified classification of error information in the future. Please accept  it.

[the remark this answers is not in the document]
Comment (dangleptr) [reply src/graph/DescribeEdgeExecutor.cpp]: Make sense.

**Comment by laura-ding** [comment]
jenkins go

**Comment by laura-ding** [comment]
close #435

**Pull request merged**